### PR TITLE
MUZIMA-687: Design and Develop mUzima Feedback feature

### PR DESCRIPTION
**Pull request by rezita**
The first drafts.

### CODE REVIEW

**Comment by samuelowino** [comment]
Hi Zita change this pull request to point to muzima:outreachy-zita branch

File: app/src/main/res/layout/activity_interactive_feedback.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                android:text="@string/feedback_header"
+                android:id="@+id/header"
+                style="@style/MessageActivityText"/>
+        <LinearLayout
+                android:orientation="vertical"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="25dp">
+            <TextView
+                    android:text="@string/feedback_rating_question1"
```
Comment (benamonya) [inline app/src/main/res/layout/activity_interactive_feedback.xml]: Hi Zita, 
Can you rename feedback_rating_question1 to feedback_rating_question

Comment (rezita) [reply app/src/main/res/layout/activity_interactive_feedback.xml]: Hello Benard, I changed it.

Comment (benamonya) [reply app/src/main/res/layout/activity_interactive_feedback.xml]: Hi @rezita, Rename the string in the other languages string.xml

Comment (rezita) [reply app/src/main/res/layout/activity_interactive_feedback.xml]: Hello @benamonya , I did it.

**Pull request merged**